install/kubernetes/cilium/values.yaml

```
@@ -267,23 +267,27 @@ bpf:
   # memory usage but can reduce latency.
   preallocateMaps: false
 
-  # -- Configure the maximum number of entries in the TCP connection tracking
+  # -- (int) Configure the maximum number of entries in the TCP connection tracking
   # table.
-  # ctTcpMax: '524288'
+  # @default -- '524288'
```

Can you change the quotes of all default values to use ` instead? It seems that way the formatting is consistent with what helm-docs does without `@default`. Thanks.

---

seems your comment missed something :) I see use `instead?`, maybe you mean remove the single quote `'`?

---

Yeah, sorry, the formatting was of. What I meant is that you replace the single quote `'` with a tick marks. For example:
```suggestion
  # @default -- `524288`
```